// try & catch statements

console.log("try & catch statements \n\n");
console.log("The try statement allows you to define a block of code to be tested for errors while it is being executed. \n\n");
console.log("The catch statement allows you to define a block of code to be executed, if an error occurs in the try block. \n\n");
console.log("\n");

console.log("Hello !!");
console.log("Hello !!");
// let a = 5;
try{
    console.log("a =", a);
} catch(err) {
    console.log("cought an error... a if not defined.");
    console.log(err);
}
console.log("Namaste !!");
console.log("Namaste !!");
console.log("Namaste !!");